README.md

```diff
@@ -247,6 +247,20 @@ Ecto.Multi.new()
 
 As you can see, the `Carbonite.Transaction` is a great place to store metadata for the operation. A `type` field can be used to categorize the transactions. A `user_id` would be a good candidate for a transaction log, as well.
 
+### Without Ecto.Multi 
+
+If you have a situation, where you want to insert a transaction without `Ecto.Multi`, then you can use `Carbonite.insert_transation/3`. Although grouping multiple operations into one transaction fits the use-case of `Ecto.Multi` perfectly so `Carbonite.Multi.insert_transaction/2` is still the preferred way here.
+
+``` elixir
+MyApp.Repo.transaction(fn ->
+  {:ok, _} = Carbonite.insert_transaction(MyApp.Repo, %{meta: %{type: "rabbit_inserted"}})
+
+  params
+  |> MyApp.Rabbit.create_changeset()
+  |> MyApp.Repo.insert!
+end)
+```
+
```

Comment:
How about removing the following section about the manual manual way of inserting a transaction through `Transaction.changeset/1`?

Reply:
Not sure, I think that is still nice to have. In the end there might be cases for this as well, who knows...

Reply:
OK, but then maybe let's remove the 

> If you don't have the luxury of an Ecto.Multi

below. Starting sentences of these two sections now make it appear that they don't know of each other